DPPP/test/unit/tPSet.cc

```diff
@@ -0,0 +1,447 @@
+// tPSet.cc: Test program for class PreFlagger::PSet
```

Comment:
Please use 'git diff -w master:DPPP/test/tPSet.cc boost-test:DPPP/test/unit/tPSet.cc' for a better diff of this file.
Main changes:
- Remove all 'using namespace xxx'. There was a name clash for cout, and the difference between casacore::Vector and std::vector is much clearer now.  The Google Style guide also forbids such using directives.
- Replace ASSERT by BOOST_CHECK
- Replace main() function by BOOST_AUTO_TEST_* statements.